monai/transforms/utils.py

```diff
@@ -368,6 +369,70 @@ def check_non_lazy_pending_ops(
         warnings.warn(msg)
 
 
+def map_and_generate_sampling_centers(
+    label: NdarrayOrTensor,
+    spatial_size: Sequence[int] | int,
+    num_samples: int,
+    label_spatial_shape: Sequence[int],
```

Comment:
```suggestion
    label_spatial_shape: Sequence[int] | None = None,
```

I think a default of None makes sense here as well as adding that to the type annotation.